src/OpenRiaServices.Tools/Test/SharedAssembliesTests.cs

```diff
@@ -217,6 +218,7 @@ public void SharedAssembliesManaged_Logs_Message_BadImageFormat_Assembly()
                 Assert.IsNull(sharedType, "Should not have detected any shared type.");
             }
 
+#if NETFRAMEWORK // Reflection Only is only supported for net framework
```

Comment:
But maybe the exception message can be created without loading the assembly, 

It seems the "message" string below does not depend on the errorMessage so it might be safe to just remove the code